ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -125,13 +125,15 @@ StatusAppTwoPanelLayout {
                         Utils.filterXSS(item.name),
                         Utils.filterXSS(item.description),
                         root.community.access,
-                        item.color.toString().toUpperCase(),
+                        item.colorString,
+                        // to retain the existing image, pass "" for the image path
                         item.image === root.community.image ? "" : item.image,
                         item.imageAx,
                         item.imageAy,
                         item.imageBx,
                         item.imageBy,
                         root.rootStore.isCommunityHistoryArchiveSupportEnabled,
+                        true // FIXME: pinMessagesAllMembers
```

Comment:
Here we need to get info about `pinMessagesAllMembers` and others flags. I don't think rootStore is good place for that. We can:
- Pass and edit whole Community object, parsed to json or just QObject-based model.
- Make editCommunity behave with empty args in way "leave it as it"

Reply:
That is a good idea! It seems out of scope for this PR, though. The best way is to create a follow-up task and tackle it there leaving this PR as a simple refactoring.

Reply:
Thanks, think in the same way.  Can i just leave here true and this FIXME? Informing @iurimatias